PhysicsTools/MVAComputer/src/VarProcessor.cc

```diff
@@ -100,8 +100,8 @@ VarProcessor *ProcessRegistry<VarProcessor, Calibration::VarProcessor,
 	if (!result) {
 		// try to load the shared library and retry
 		try {
-			delete VPPluginFactory::get()->create(
-					std::string("VarProcessor/") + name);
+			std::unique_ptr<PhysicsTools::VarProcessor::Dummy> tmp {VPPluginFactory::get()->create(
+					std::string("VarProcessor/") + name)};
 			result = ProcessRegistry::create(name, calib, parent);
 		} catch(const cms::Exception &e) {
```

Comment:
Here is a similar pattern as in https://github.com/cms-sw/cmssw/pull/25914/files#r256024015, except
- the code rethrows (so should use `create()`
- the code issues a LogError before rethrowing, so needs the try-catch (although I'm not so sure about the usefulness of the LogError if the exception propagates to the framework)